README.md

```diff
@@ -17,35 +17,67 @@ Muchos is structured into two high level components:
 
  * [Ansible] scripts that install and configure Fluo and its dependencies on a cluster.
  * Python scripts that push the Ansible scripts from a local development machine to a cluster and
-   run them. These Python scripts can also optionally launch a cluster in EC2 using [boto].
+   run them. These Python scripts can also optionally launch a cluster in EC2 using [boto] or in Azure using Azure CLI.
 
 Checkout [Uno] for setting up Accumulo or Fluo on a single machine.
 
-## Requirements
+## Requirements 
 
-Muchos requires the following:
+### Common
+
+Muchos requires the following common components for installation and setup:
+
+* Python 3.6.8 with a virtual environment setup
+Create a Python environment and switch to it
+```bash
+cd ~
+python3.6 -m venv env
+source env/bin/activate
+```
+* `ssh-agent` installed and running and ssh-agent forwarding.  Note that this may also require the creation of SSH public-private [key pair](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/mac-create-ssh-keys).
+```bash
+eval $(ssh-agent -s)
+ssh-add ~/.ssh/id_rsa
+```
+* Git (current version)
+
+### EC2
+
+Muchos requires the following for EC2 installations:
 
-* Python 3
 * [awscli] & [boto3] libraries - Install using `pip3 install awscli boto3 --upgrade --user`
-* `ssh-agent` installed and running
 * An AWS account with your SSH public key uploaded. When you configure [muchos.props], set `key.name`
   to name of your key pair in AWS.
 * `~/.aws` [configured][aws-config] on your machine. Can be created manually or using [aws configure][awscli-config].
 
+### Azure
+
+Muchos requires the following for Azure installations:
+
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)  must be installed, configured and authenticated to an Azure subscription. Please note - you should install [Azure CLI 2.0.69](https://packages.microsoft.com/yumrepos/azure-cli/azure-cli-2.0.69-1.el7.x86_64.rpm) on CentOS. Higher versions of Azure CLI are  unsupported for Muchos on CentOS at this time. Example command to install Azure CLI 2.0.69 on CentOS is below:
```

Comment:
@keith-turner, it is CentOS 7, will update the ReadME with specific version of CentOS. As far as specific version of Azure CLI  (2.0.69), it is only applicable to CentOS at the moment. There is an #[issue logged ](https://github.com/Azure/azure-cli/issues/10128) with Azure CLI.  On rest, latest version of Azure CLI (2.0.70) is applicable

Reply:
To add, usage of CentOS is just one option. We have also executed this with Ubuntu and it works fine with the same

Reply:
> it is only applicable to CentOS at the moment. 

That is what I was thinking when I said it would become stale.  Would there be a bug that could also be referenced so if it does become stale people would know this by clicking on the bug and seeing it was fixed?

Reply:
Nevermind the bug was mentioned in an earlier comment, that could be referenced in the readme.

Reply:
Great, we have added that note to the README